report/src/sections/02-preliminaries.tex

```diff
@@ -129,43 +128,28 @@ \subsection{Initialization of HMM Parameters}\label{subsec:initialization}
-The initial state probabilities $\pi$ represent the likelihood of starting in each hidden state $s \in S$.
-To estimate initial state probabilities, we can use one of the following strategies:
-
-\begin{itemize}
-    % \item Examine the observed data: Estimate the initial probabilities based on the observed data. For example, if certain states are more likely to occur at the beginning of the sequence, assign higher probabilities to these states. 
-    % $\pi_s = \frac{R_s}{\sum_{s' \in S} R_{s'}}$ where $R_s$ is the number of times state $s$ occurs at the beginning of the sequence.
-    \item Random initialization: Assign random probabilities to each state $s \in S$, such that $\sum_{s \in S} \pi_s = 1$.
-    \item Uniform initialization: Set $\pi_s = \frac{1}{|S|}$ for all $s \in S$.
-\end{itemize}
-
-\textit{Initialization transition probability distribution}:
-
-The transition matrix $P$ represents the probability of transitioning from one state to another.
-To estimate transition probabilities, we can use one of the following strategies:
-\begin{itemize}
-    % \item Examine the observed data: Estimate the transition probabilities based on the observed data. For example, if certain states are more likely to transition to others, assign higher probabilities to these transitions. 
-    % $P_{s \rightarrow s'} = \frac{C_{s \rightarrow s'}}{\sum_{s'' \in S} C_{s \rightarrow s''}}$ where $C_{s \rightarrow s'}$ is the number of transitions from state $s$ to state $s'$.
-    \item Random initialization: Assign random probabilities to each transition $P_{s \rightarrow s'}$, such that $\sum_{s' \in S} P_{s \rightarrow s'} = 1$.
-    \item Uniform initialization: Set $P_{s \rightarrow s'} = \frac{1}{|S|}$ for all $s, s' \in S$.
-\end{itemize}
-
-\textit{Initialization emission probability distribution}:
-
-The emission matrix $\omega$ represents the probability of emitting each observation label from each hidden state.
-To estimate emission probabilities, we can use one of the following strategies:
-\begin{itemize}
-    % \item Examine the observed data: Estimate the emission probabilities based on the observed data. For example, if certain states are more likely to emit certain labels, assign higher probabilities to these emissions. 
-    % $\omega_{s, l} = \frac{C_{s, l}}{\sum_{l' \in \mathcal{L}} C_{s, l'}}$ where $C_{s, l}$ is the number of times state $s$ emits label $l$.
-    \item Random initialization: Assign random probabilities to each emission $\omega_{s, l}$, such that $\sum_{l \in \mathcal{L}} \omega_{s, l} = 1$ for all $s \in S$.
-    \item Uniform initialization: Set $\omega_{s, l} = \frac{1}{|\mathcal{L}|}$ for all $s \in S, l \in \mathcal{L}$.
-\end{itemize}
-
-We initialize the parameters using uniform initialization for the emission matrix $\omega$, the transition matrix $P$, and the initial state distribution $\pi$.
+
+As we are working with \glspl{hmm} we dont have any prior knowledge of the model parameters.
```

Comment:
Is this the reason? Is the reason not rather that we're dealing with an unknown model for a number of observations?